BREAKING-CHANGES.md

```diff
@@ -47,6 +49,14 @@ This is a quick summary of the sections below:
   - MSRV is now 1.63.0
   - `List` no longer ignores empty strings
 
+## [v0.26.1](https://github.com/ratatui-org/ratatui/releases/tag/v0.26.1)
```

Comment:
Is this the correct version for this - likely 'unreleased' or 0.26.3

Reply:
Ugh my bad.